### PR TITLE
Update Auto-Updator

### PR DESCRIPTION
**Pull request by GamingLiamStudios**
The auto-update feature now fetches the newest build from circle-ci.
You can also change the branch which said builds will come from.

And of course, OWO translation for it all.
oh, and vscode doing a bunch of formatting, as usual.

### CODE REVIEW

**Comment by coredev-uk** [comment]
Please revert the formatting if possible, its very hard to track the changed that you've made and the formatting that you've done will most likely get reverted.

**Comment by coredev-uk** [comment]
I'll leave this as a draft. Change it back when you are ready.

**Comment by GamingLiamStudios** [comment]
I've gone through and fixed a lot of the formatting that vscode fucked up. Still quite a bit of changes tho, but it's 2.10am, they look nice and I just want to sleep.

**Comment by GamingLiamStudios** [comment]
General rundown of what I did do:
- Added new dropdown menu under 'Update Cider' button to choose which branch to update from
- Added circle-ci API call to get most recent build from given branch
- Replaced `term.updateCider` to `settings.option.experimental.updateCider`
- OWO translation of all the new settings
- Some formatting(blame vscode. except for i18n files, that was me.)

File: src/renderer/views/pages/settings.ejs
Comment (quacksire) [inline src/renderer/views/pages/settings.ejs:661]: keep this. for vue

**Review by quacksire — CHANGES_REQUESTED** [review]

**Comment by GamingLiamStudios** [comment]
Shit. Don't merge, somehow broke cider. Probably with that upstream merge.

**Comment by GamingLiamStudios** [comment]
And fixed. was a stray bracket.

**Comment by GamingLiamStudios** [comment]
Also, before we do merge this, I want to propose a new feature for the Auto-Updater. A popup saying the status of the update, such as `Cider is already up to date` or `Your platform does not support Auto-Updates`
I'll look into how this might work, as it would make Updating feel a bit more responsive.

**Review by quacksire — APPROVED** [review]

**Pull request merged**
